icu4j/main/core/src/main/java/com/ibm/icu/segmenter/LocalizedSegmenter.java

```diff
@@ -31,7 +31,7 @@ public static Builder builder() {
     return new Builder();
   }
 
-  private LocalizedSegmenter(ULocale locale, SegmentationType segmentationType) {
+  private LocalizedSegmenter(SegmentationType segmentationType, ULocale locale) {
```

Comment:
for these parameters I prefer locale then type like you had it...

my comment was about the LocalizedSegments constructor -- which you swapped in your last commit today -- not this constructor here